Graphs/KannsAlgorithm.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * @author {RaviSadam}
+ * @name kannsAlgorithm
+ * @description -
+ * Kann's Algorithm implementation in JavaScript
+ * @summary
+ * Kann's Algorithm is used for topological sorting in directed acyclic graphs
+ *
+ * @param graph - Graph [[v1,v2],[v3,v4,v5]..]
+ * @param n - number of vertices
+ * @returns {Array} - Empty array if cycle is detected or else result array;
+ *
+ */
+
+export function kannsAlgorithm(graph, n) {
+  if (n === null || n === undefined) throw Error('Invalid n was given')
+  const inorder = Array(n).fill(0)
+  const result = []
+  for (let entry of graph) {
+    for (let edge of entry) {
+      inorder[edge] += 1
+    }
+  }
+  const queue = []
```

Comment:
This need not be a queue. I recommend using it as a stack instead.

A better name would also inform readers that these are the nodes with no incoming edge.

Reply:
I am using BFS algo. So have to use Queue data structure